### PR TITLE
Add MIT License section to optimize.py, state.py, workers.py,

### PR DESCRIPTION
**Pull request by ddyy345**

### CODE REVIEW

File: msibi/optimize.py [hunk 1]
```diff
@@ -1,3 +1,34 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
```
Comment (tcmoore3) [inline msibi/optimize.py]: Both Christoph and I are authors.

File: msibi/pair.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors: Timothy C. Moore, Davy Yue
```
Comment (tcmoore3) [inline msibi/pair.py]: Both Christoph and I are authors.

File: msibi/optimize.py [hunk 3]
```diff
@@ -1,3 +1,34 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
```
Comment (tcmoore3) [inline msibi/optimize.py]: Vanderbilt University

File: msibi/optimize.py [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+#
+# MSIBI IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH MSIBI OR THE USE OR OTHER DEALINGS IN THE
+# MSIBI.
+#
+# You should have received a copy of the MIT license.
```
Comment (tcmoore3) [inline msibi/optimize.py]: ...along with MSIBI.

File: msibi/potentials.py [hunk 5]
```diff
@@ -1,3 +1,33 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors: Timothy C. Moore
```
Comment (tcmoore3) [inline msibi/potentials.py]: Both Christoph and I are authors.

File: msibi/state.py [hunk 6]
```diff
@@ -1,3 +1,33 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors: Timothy C. Moore, Davy Yue
```
Comment (tcmoore3) [inline msibi/state.py]: Both Christoph and I are authors.

File: msibi/workers.py [hunk 7]
```diff
@@ -1,3 +1,33 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+#   iterative Boltzmann inversion.
+# Copyright (c) 2017 MIT University and the Authors
+#
+# Authors: Christoph Klein
```
Comment (tcmoore3) [inline msibi/workers.py]: Both Christoph and I are authors.

File: msibi/optimize.py (hunk 1 above)
Comment (ddyy345) [inline msibi/optimize.py]: Sorry about that. I changed the Authors and Collaborators section according to the format in the setup.py file. Will update to reflect requests.

**Comment by ctk3b** [comment]
Thanks!

**Pull request merged**